### ISSUE TITLE
Overview of Kit Structures of mmdetection

### ISSUE THREAD

**Issue by Keiku** [open-mmlab/mmdetection#6868]
**Describe the feature**

Since the Overview of Kit Structures of Paddle Detection is easy to understand, how about putting the mmdetection version of it in Document or README.md?

**Motivation**

To make it easier to see what mmdetection supports

**Related resources**

PaddlePaddle/PaddleDetection: Object Detection toolkit based on PaddlePaddle. It supports object detection, instance segmentation, multiple object tracking and real-time multi-person keypoint detection. https://github.com/PaddlePaddle/PaddleDetection#overview-of-kit-structures

**Comment by hhaAndroid** [comment]
@Keiku This is a good idea, we will discuss it as soon as possible.

**Comment by Keiku** [comment]
@hhaAndroid I tried to make a similar list, but I couldn't list it well. I think everyone will be helpful if the official support is available. Thank you for your consideration.

[Keiku/mmdet-overview-of-kit-structures: Overview of Kit Structures of mmdetection](https://github.com/Keiku/mmdet-overview-of-kit-structures)

**Comment by hhaAndroid** [comment]
@Keiku This is very nice, but it would be better if it can be made into a table format. Once you are done, you can create a PR and we will review it. Thank you.

**Comment by Keiku** [comment]
@hhaAndroid It was a tough task, but I converted it to table format. Summarizing the implementation was difficult, so I decided to summarize model zoo (configs). I think it would be better to format of the model zoo page of mmdetection's official README.md to the table format.

The comment of `#Not officially listed` is an item that is not listed in the official README.md, so I comment it just in case. If you check the links below and it looks good, I would like to make a PR.

Overview of Benchmark and Model Zoo
https://github.com/Keiku/mmdet-overview-of-kit-structures/blob/main/model_zoo_table.md
